src/util/util.ts

```diff
@@ -36,11 +36,28 @@ export async function specifyTargetFolder(uri: vscode.Uri): Promise<vscode.Uri>
     return fileUri;
 }
 
-export function getPackagePath(srcDir: string): string {
-    var index = srcDir.indexOf("java");
-    index = index + 5;
-    var packageName = srcDir.substring(index);
-    return packageName;
+export async function getPackageName(srcDir: string, pathSep: string): Promise<string> {
+    // looking for "/java/" from the path
+    var index = srcDir.toLowerCase().indexOf(pathSep + "java" + pathSep);
+    let defaultPackageName: string;
+    if (index > -1) {
+        index = index + 6;
+        defaultPackageName = srcDir.substring(index).replace(pathSep, "."); // use everything after the "/java/" to be the package name
+    }
+    
+    var exp = /^[a-z][a-z0-9_]*(\.[a-z0-9_]+)*$/;   // validate the package name
```

Comment:
When validating the package name, the default name does not replace all of the `/` with `.` . The user is then unable to submit the auto-filled package name until they manually replace the `/` with `.`
![image](https://user-images.githubusercontent.com/26146482/72901271-770ab300-3cf7-11ea-9070-b72ddcd30cd7.png)

Reply:
Fixed the "replace" issue, please give it a try.